server.go

```diff
@@ -739,7 +753,11 @@ func (s *server) handleOpenChanReq(req *openChanReq) {
 	// of blocking on this request which is exported as a synchronous
 	// request to the outside world.
 	// TODO(roasbeef): server semaphore to restrict num goroutines
-	go s.fundingMgr.initFundingWorkflow(targetPeer, req)
+	peerAddress := &lnwire.NetAddress{
+		Address:     targetPeer.addr.Address,
+		IdentityKey: req.targetPubkey,
+	}
+	go s.fundingMgr.initFundingWorkflow(peerAddress, req)
```

Comment:
Similar to the comment above, the peer's existing `lnwire.NetAddres` instance can be passed in here rather than creating a new one.